### DISCUSSION TITLE
How to change titlebar height

### DISCUSSION THREAD

**Discussion by vesal** [nextapps-de/winbox#133] [Ideas]
For mobile I like to have smaller titlebar.

I tried with following css:

```
.wb-body {
    top: 15px;
} 
.wb-icon {
    height: 15px;
}   
.wb-title {
    line-height: 15px;
}
```

It almost worked, but when minimized, the element height is still fixed 35px so that is somewhere hard coded?

**Comment by mossaiby** [comment]
I see the height is set in element `style`. Searching for `35` in the JS source, I think it is hard-coded 
here:

https://github.com/nextapps-de/winbox/blob/a74fbd6e7fcb5f6270caee016fd15a5249904834/src/js/winbox.js#L378

Hope this helps.

**Comment by ts-thomas** [comment]
Custom title bar height is now supported

**Reply by xznhj8129** [reply]
Could you indicate where or how that's done? There's no clear way shown to do it, trying to resize and align the individual parts of the title bar just leads to chaos
